e2e/pageobjects/openshift/ILoginPageOcp.ts

```diff
@@ -0,0 +1,13 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+export interface ILoginPageOcp {
+    login(): void;
```

Comment:
On my opinion this interface should be removed. Instead "OpenShiftLoginByTempAdmin" and "OpenShiftLoginPage" should be rebinded to the "ILoginPage" interface and moved to the login folder.

Reply:
+1

Reply:
Yeah.. You're right.. I'll rework it.